frontend/src/libs/components/form/form.tsx

```diff
@@ -18,6 +21,37 @@ type Properties<T extends FieldValues> = {
   onSubmit: (payload: T) => void;
 };
 
+function renderField<T extends FieldValues = FieldValues>(
```

Comment:
why not arrow function?

Reply:
yeah, it's a little bit strange here, re-write